error-handling.js

```diff
@@ -0,0 +1,11 @@
+const db = require("./db/connection");
+
+exports.errorHandler500 = (err, request, response, next) => {
+  console.log(error);
+  response.status(500).send({ msg: "Server error" });
+};
+
+exports.errorHandler400 = (err, request, response, next) => {
+  console.log(error);
+  response.status(400).send({ msg: "Bad request" });
+};
```

Comment:
Could we change the 400 to a 404 please? This would be more appropriate at this stage 🙂